gammapy/modeling/models/temporal.py

```diff
@@ -170,18 +170,70 @@ def integral(self, t_min, t_max):
         return (t_max - t_min) / self.time_sum(t_min, t_max)
 
 
+class LinearTemporalModel(TemporalModel):
+    r"""Temporal model with a linear variation.
+
```

Comment:
Add a forward reference to the example plot here, like `For more information see :ref:`linear-temporal-model`.`